lib/widgets/details_widget.dart

```diff
@@ -0,0 +1,120 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:marvel/bloc/details_bloc.dart';
+import 'package:marvel/bloc/marvel_state.dart';
+import 'package:marvel/data/model/character.dart';
+import 'package:marvel/data/model/series.dart';
+import 'package:marvel/widgets/my_image.dart';
+
+class DetailsWidget extends StatelessWidget {
+  final String description;
+  final Thumbnail thumbnail;
+
+  const DetailsWidget(
+      {Key? key, required this.description, required this.thumbnail})
+      : super(key: key);
+
+  String _createThumbnailUrl(Thumbnail thumbnail) {
+    return thumbnail.path + '/landscape_xlarge.' + thumbnail.extension;
+  }
+
+  String _createThumbnailUrlSeries(Series item) {
+    return item.thumbnail.path + '/portrait_medium.' + item.thumbnail.extension;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocBuilder<DetailsBloc, MarvelState>(
+      builder: (context, state) {
+        if (state is DataLoadingState) {
+          return const Center(
+            child: CircularProgressIndicator(),
+          );
+        }
+        if (state is DetailsLoadedState) {
+          return SingleChildScrollView(
```

Comment:
Widget is too big, can we extract to the smaller parts via Widgets or private methods? 
Here is a good example: [link](https://github.com/alexsit19/marvelapp_flutter/pull/1/files#diff-d5931a3e4569ec2e0c7b245e08ab8ffc4bb9ff47846239d9accae1f016c96a2cR25)